textalyze.js

```diff
@@ -14,10 +14,32 @@
 function itemCounts(array) {
   let counts = new Map();
 
+  array.forEach(item => {
+
+    if (!counts.has(item)) {
+      counts.set(item, 0);
+    }
+
+    counts.set(item, parseInt(counts.get(item)) + 1);
+
+  });
+
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
   return counts;
 }
 
+let args = "";
```

Comment:
This code is meant to both be included as a module (via `require`) and run directly (via `npm start`).

To ensure that this code will run _only_ when `textalyze.js` is run directly, do the following: https://gist.github.com/miguelmota/a6258318a5b7ce9b462243e25c2025a5

So, it'd look like:

```javascript
if (require.main === module) {
  let args = "aabbccd";
  console.log(`...`);
  // etc. etc.
 
}
```

This will prevent the code from being run when `textalyze.js` is loaded via `require`.